src/components/data-management/SamplesTableCells.jsx

```diff
@@ -166,7 +168,7 @@ const SampleNameCell = (props) => {
   const { text, record, idx } = cellInfo;
   const dispatch = useDispatch();
   return (
-    <Text strong key={`sample-cell-${idx}`}>
+    <Text className={integrationTestConstants.classes.SAMPLE_NAME_CELL} strong key={`sample-cell-${idx}`}>
```

Comment:
Regarding to comment in https://github.com/biomage-ltd/testing/pull/18, maybe change to `SAMPLES_TABLE_NAME_CELL`?